src/components/WebexAvatar/WebexAvatar.js

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import {Avatar} from '@momentum-ui/react';
 
-import usePerson from '../hooks/usePerson';
+import {usePerson, useAdapter} from '../hooks';
 
 export default function WebexAvatar(props) {
-  const {personID, adapter} = props;
-  const {avatar, displayName, status} = usePerson(personID, adapter);
+  const {personID} = props;
+  const adapter = useAdapter();
+  const {avatar, displayName, status} = usePerson(personID, adapter.peopleAdapter);
 
   return <Avatar src={avatar} title={displayName} type={status} alt={displayName} />;
 }
 
 WebexAvatar.propTypes = {
   personID: PropTypes.string.isRequired,
```

Comment:
Remove trailing comma, please
```suggestion
  personID: PropTypes.string.isRequired
```